docs/resources/obs_bucket_notification.md

```diff
@@ -0,0 +1,68 @@
+---
+subcategory: "Object Storage Service (OSS)"
+description: ""
+page_title: "flexibleengine_obs_bucket_notification"
+---
+
+# flexibleengine_obs_bucket_notification
+
+Manages an OBS bucket **Notification Configuration** resource within FlexibleEngine.
+
+[Notification Configuration](https://docs.prod-cloud-ocb.orange-business.com/usermanual/obs/en-us_topic_0045853816.html)
+OBS leverages SMN to provide the event notification function. In OBS, you can use SMN to send event notifications to
+specified subscribers, so that you will be informed of any critical operations (such as upload and deletion) 
+that occur on specified buckets in real time.
+
+## Example Usage
+
+### OBS Notification Configuration
+
+```hcl
+resource "flexibleengine_obs_bucket" "bucket" {
+  bucket = "my-test-bucket"
+  acl = "public-read"
+}
+
+resource "flexibleengine_obs_bucket_notification" notification {
+  bucket = flexibleengine_obs_bucket.bucket.bucket
+  topic_configurations {
+    name      = "notification_name"
+    events    = ["ObjectRemoved:*"]
+    prefix    = "tf_update"
+    suffix    = ".png"
+    topic_urn = "urn:smn:eu-west-0:d8cb0fdcf29b4badb9ed8b2525a3286f:topic"
+  }
+}
```

Comment:
the example is not correct, there is no topic_configurations block

Reply:
OK, i have fixed it